pysatModels/tests/test_utils_testing.py

```diff
@@ -25,9 +24,7 @@ def setup_method(self):
 
         # Load the data in the instruments
         load_kwargs = {'date': pysat_testmodel._test_dates['']['']}
-        if(pack_version.Version(pysat.__version__)
-           > pack_version.Version('3.0.1')):
-            load_kwargs['use_header'] = True
+        load_kwargs['use_header'] = True
```

Comment:
Recommend keeping the version test, but add a max version for this as well.  

```
if all([Version(pysat.__version__) > Version('3.0.1'),
        Version(pysat.__version__) < Version('3.2.0')]):
    inst_kwargs['use_header'] = True
```

Reply:
Since the requirements files specifies a minimum of pysat 3.0.4, should we just proceed with the maximum cap?

Reply:
🤒 excellent point